### ISSUE TITLE
Programming Exercise shows wrong preview

### ISSUE THREAD

**Issue by Garfield96** [ls1intum/Artemis#966]
### Describe the bug
The description field for programming exercises doesn't show the correct preview in the following two cases:

- [ ] **1. Bug**
##### To Reproduce:
Enter the following markdown
```
# Test
1. [task][Task Short Description](testCaseName){hintId1, hintId2}
2. [task][Task Short Description](testCaseName){hintId1, hintId2}
```
##### Result:
![image](https://user-images.githubusercontent.com/15248220/67191052-444bd880-f3f1-11e9-90a9-677ccc53c4ce.png)
##### Expected behavior:
Two distinct lines.

---

- [x] **2. Bug**
##### To Reproduce:
Enter a long list of tasks
##### Result:
![image](https://user-images.githubusercontent.com/15248220/67192283-97bf2600-f3f3-11e9-90a8-64005e2e73a0.png)
##### Expected behavior:
Banner shouldn't be transparent.

### Environment
<details><pre>
 - OS: Debian Buster
 - Browser: Mozilla Firefox
 - Version: 60.9.0esr (64-bit)
</pre></details>

**Comment by thilo-behnke** [comment]
**Concerning 1:**
This is a bug / unhandled case, if tasks are exactly the same, only the first one will receive rendered html.
The workaround is to at least use different task names:

```
# Test
1. [task][Task Short Description](testCaseName){hintId1, hintId2}
2. [task][Task Short Description2](testCaseName){hintId1, hintId2}
```

**Concerning 2:**
I remember we recently had a similar issue but thought we fixed it.
I also tried reproducing the issue with a list of 200 tasks.
If the issue still persists, can you provide the markdown you used?

**Comment by krusche** [comment]
@Garfield96 you are using Firefox right?
Could this be a browser specific issue?

I just tried it in Chrome and Safari and could not reproduce the text overlay at the bottom.
What happens in a different browser?

**Comment by thilo-behnke** [comment]
@Garfield96 @krusche I can reproduce it on Firefox.

**Comment by madwau** [comment]
I wrote a fix for the unwanted overflow in #1102.

**Comment by madwau** [comment]
I think we agreed that bug 1 does not occur in normal use cases and is therefore not a priority.

Bug 2 has been fixed in #1102.
